invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,6 +36,241 @@ def create_vocabulary_record(service_str, data):
 @shared_task
 def create_demo_record(data):
     """Create a demo record."""
-    service = current_rdm_records.records_service
+    service = current_rdm_records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task
+def create_demo_community_submission_request(
+        user_id,
+        community_data,
+        drafts_data,
+        n_drafts=1,
+        n_requests=1
+):
+    """Create n_requests community_submission requests for n_drafts drafts."""
+    """Drafts are created in a new community."""
+    # Create a demo community
+    user = current_datastore.get_user(user_id)
+    community = current_communities.service.create(
+        user,
+        community_data
+    )
+    Community.index.refresh()
+    records_service = current_rdm_records_service
+
+    # Create drafts in the community
+    for i in range(n_drafts):
+        _create_draft.delay(
+            user_id,
+            community,
+            records_service,
+            drafts_data[i],
+            n_requests,
+        )
+
+
+@shared_task
+def _create_draft(
+        user_id,
+        community,
+        records_service,
+        draft_data,
+        n_request,
+):
+    # Create a demo draft
+    user = current_datastore.get_user(user_id)
+    draft_data['parent'] = {
+        'review': {
+            'type': 'community-submission',
+            'receiver': {'community': community.data['uuid']}
+        }
+    }
+
+    draft = records_service.create(
+        data=draft_data, identity=user
+    )
+
+    # Create requests for the draft
```

Comment:
Please be more explicit:
```suggestion
    # Create a new request to incude a draft to a community
```